ui/uihandler.go

```diff
@@ -513,16 +501,24 @@ func Serve(port int, debugpath string, uipath string) {
 		debugPath = ".." + debugpath // TODO: calculate actual path if not same number of directories
 		http.HandleFunc(uiPath, Handler)
 		fmt.Printf("UI starting - visit:\nhttp://localhost:%d%s\n", port, uiPath)
-		logoPath = uiPath + logoURI
-		http.HandleFunc(logoPath, LogoHandler)
-		chartJSPath = uiPath + chartjsURI
-		http.HandleFunc(chartJSPath, ChartJSHandler)
-		// Use relative paths after that (in the html template):
-		logoPath = "./" + logoURI
-		chartJSPath = "./" + chartjsURI
+
 		fetchPath = uiPath + fetchURI
 		http.HandleFunc(fetchPath, FetcherHandler)
 		fhttp.CheckConnectionClosedHeader = true // needed for proxy to avoid errors
 	}
+	logoPath = "../static/img/logo.svg"
+	chartJSPath = "../static/js/Chart.min.js"
+
+	// Serve static contents in the ui/static dir.
+	// We use directory relative to this file to find the static contents,
+	// so no matter where the generate go binary is, the static dir could be found.
+	_, filename, _, ok := runtime.Caller(0)
```

Comment:
it can probably wait the next pr / when we make packages but we should have a variable for that directory and only fall back to caller if not set (we would set that variable when making a package)